core/block_svg.js

```diff
@@ -592,6 +595,63 @@ Blockly.BlockSvg.prototype.clearTransformAttributes_ = function() {
 };
 
 /**
+ * Transforms a block by setting the translation on the transform attribute
```

Comment:
Looking at `develop` this doesn't look like the first time we've had the problem: I see three definitions for `Blockly.BlockSvg.prototype.translate`.